Formula/t-rec.rb

```diff
@@ -0,0 +1,23 @@
+class TRec < Formula
+  desc "Blazingly fast terminal recorder that generates animated gif images for the web"
+  homepage "https://github.com/sassman/t-rec-rs"
+  url "https://github.com/sassman/t-rec-rs/archive/v0.2.1.tar.gz"
+  sha256 "a23ae0e19b76740220f4640a82d2813fe1002609bbf28ab180f3fc9228b141e3"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
+  depends_on "imagemagick"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    # let's fetch the window id
+    o = `#{bin}/t-rec -l | tail -1`
+    win_id = o.split(/\s|\n/)[-1]
+    # verify that it's an appropriate id
+    raise "No window id retrieved" unless win_id && Integer(win_id).positive?
```

Comment:
No need to raise in tests, please use assertions instead.

Reply:
adjusted.